lib/kb_uploadmethods/kb_uploadmethodsImpl.py

```diff
@@ -176,6 +176,11 @@ def upload_fasta_gff_file(self, ctx, params):
         print('--->\nRunning uploadmethods.upload_fasta_gff_file\nparams:')
         print((json.dumps(params, indent=1)))
 
+        if params.get('ncbi_taxon_id'):
+            params['taxon_id'] = params['ncbi_taxon_id']
+        if params.get('relation_engine_timestamp_ms'):
```

Comment:
Do you want users to input a valid epoch time in UI? I am not sure how this will be used in GFU. Shouldn't it be generated by `time.time()`?

Reply:
I have added the {"narrative_system_variable": "timestamp_epoch_ms"} to pass the epoch time in ui.  GFU will default to time.time() if the time_stamp is not passed.

Reply:
why not `{"narrative_system_variable": "time_stamp"}`? The assingment `ncbi_taxon_id` -> `taxon_id` and `relation_engine_timestamp_ms` -> `time_stamp` makes no sense to me.